using-timescaledb/compression.md

```diff
@@ -305,15 +306,16 @@ to re-compress the chunks that we are currently working on (not the desired resu
 To accomplish this we first find the job_id of the policy using:
 
 ```sql
-SELECT job_id, table_name
-FROM _timescaledb_config.bgw_policy_compress_chunks p
-INNER JOIN _timescaledb_catalog.hypertable h ON (h.id = p.hypertable_id);
+SELECT s.job_id
+FROM timescaledb_information.jobs j
+INNER JOIN timescaledb_information.job_stats s ON j.job_id = s.job_id
+WHERE j.proc_name = 'policy_compression' AND s.hypertable_name = <target table>;
```

Comment:
```suggestion
SELECT s.job_id
  FROM timescaledb_information.jobs j
  INNER JOIN timescaledb_information.job_stats s ON j.job_id = s.job_id
  WHERE j.proc_name = 'policy_compression' AND s.hypertable_name = <target table>;
```

Reply:
@mfreed Your proposal is not consistent with other examples in this and other docs. The top select clauses are usually indented in the same way as `SELECT`, however the join should have an additional indentation. I believe it should be:
```suggestion
SELECT s.job_id
FROM timescaledb_information.jobs j
  INNER JOIN timescaledb_information.job_stats s ON j.job_id = s.job_id
WHERE j.proc_name = 'policy_compression' AND s.hypertable_name = <target table>;
```
or
```suggestion
SELECT s.job_id
FROM timescaledb_information.jobs j
     INNER JOIN timescaledb_information.job_stats s ON j.job_id = s.job_id
WHERE j.proc_name = 'policy_compression' AND s.hypertable_name = <target table>;
```

Reply:
Doing a quick scan of the docs, it looks like there isn't a standard usage here.  Most common appears to be having nothing indented, though there are some examples where only joins are indented and a few others where everything is indented.  Personally I find Ruslan's suggestion a bit more readable than Mike's here (though I find the larger indentation a bit glaring), so I'm going with that for the moment.  I expect we'll have to revisit this if we decide to standardize our docs sql.

Reply:
Looking at that example doc, it does look like there's a effort to make them more readable.  However, there still doesn't seem to be much consistency between examples.